### PR TITLE
docs: Create AsyncSetMuteUndefined sample

### PR DESCRIPTION
**Pull request by hannahtsai2024**
Add AsyncSetMuteUndefined.java code sample for use case: Resetting an individual finding mute_state to UNDEFINED.

### CODE REVIEW

File: java-securitycenter/samples/snippets/generated/com/google/cloud/securitycenter/v2/securitycenter/setmute/AsyncSetMuteUndefined.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+import com.google.cloud.securitycenter.v2.SetMuteRequest;
+
+public class AsyncSetMute {
+
+  public static void main(String[] args) throws Exception {
+    asyncSetMute();
+  }
+
+  public static void asyncSetMute() throws Exception {
```
Comment (iennae) [inline java-securitycenter/samples/snippets/generated/com/google/cloud/securitycenter/v2/securitycenter/setmute/AsyncSetMuteUndefined.java:33]: you need to update the comments in the comments to explain what is happening; while the source material may be auto-generated, if you've modified this, you should make it clear and specific to the context of the snippet.

Comment (hannahtsai2024) [reply java-securitycenter/samples/snippets/generated/com/google/cloud/securitycenter/v2/securitycenter/setmute/AsyncSetMuteUndefined.java]: Acked, modified the comments, class name and fixed imported Mute class version.

**Comment by iennae** [comment]
@hannahtsai2024 I didn't notice this is editing samples in the existing generated samples. This isn't the java docs samples repo so my edit is entirely incorrect https://github.com/GoogleCloudPlatform/java-docs-samples is where you would submit this PR not to this repository. This is where all the generated samples live.

**Comment by hannahtsai2024** [comment]
> @hannahtsai2024 I didn't notice this is editing samples in the existing generated samples. This isn't the java docs samples repo so my edit is entirely incorrect https://github.com/GoogleCloudPlatform/java-docs-samples is where you would submit this PR not to this repository. This is where all the generated samples live.

Got it, I created a new PR for this sample. Thanks for pointing this out! https://github.com/GoogleCloudPlatform/java-docs-samples/pull/9443

**Pull request closed without merging**